app/cafe.py

```diff
@@ -0,0 +1,24 @@
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError,
+                        VaccineError)
+import datetime
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        try:
+            if "vaccine" not in visitor:
+                raise NotVaccinatedError("NotVaccinatedError")
+            elif visitor["vaccine"]["expiration_date"] < datetime.date.today():
+                raise OutdatedVaccineError("OutdatedVaccineError")
+            elif not visitor["wearing_a_mask"]:
+                raise NotWearingMaskError("NotWearingMaskError")
+            return f"Welcome to {self.name}"
+        except VaccineError:
+            raise
+        except NotWearingMaskError:
+            raise
```

Comment:
The exception handling for `VaccineError` is redundant because `VaccineError` is a superclass of `NotVaccinatedError` and `OutdatedVaccineError`. Since these specific exceptions are already being raised, catching `VaccineError` separately is unnecessary. You can remove the `except VaccineError` block to simplify the code.